src/user-research/how-we-do-user-research.md

```diff
@@ -164,9 +167,8 @@ ethical, inclusive and sustainable ways.
 We want to involve our colleagues and other stakeholders in creating research
 plans, so they understand and can contribute to research activities.
 
-And we want to create plans that allow us to
-[learn, share and adapt](https://playbook.dxw.com/#user-research-principles) as
-we go.
+And we want to create plans that allow us to [learn, share and adapt](..) as we
```

Comment:
Broken link?

Reply:
That's a fully functioning relative link, isn't it?

Reply:
You're right!